routes/dev.php

```diff
@@ -125,3 +125,6 @@
     return MinecraftApiService::playerUsernameToUuid('xinecraft');
 });
 
+Route::get('app-key', function () {
+    return config('app.key');
```

Comment:
Check regarding the dev.php changes. Can this be security issue?

Reply:
Check regarding the dev.php changes. Can this be security issue? @openai

Reply:
what?

Reply:
How you came to that conclusion that app key is harmless to expose? Is appkey or a laravel app harmless to expose to public?

Reply:
you are wrong. App key should never be exposed.

Reply:
yea. now suggest change to not expose API key. remove the route.

Reply:
now suggest change to not expose API key. remove the route.

Reply:
ok remove the route